archunit/src/main/java/com/tngtech/archunit/library/plantuml/rules/PlantUmlPatterns.java

```diff
@@ -36,17 +37,20 @@
 class PlantUmlPatterns {
     private static final String COMPONENT_NAME_GROUP_NAME = "componentName";
     private static final String COMPONENT_NAME_FORMAT = "\\[" + capture(anythingBut("\\[\\]"), COMPONENT_NAME_GROUP_NAME) + "]";
-
+    private static final String DATABASE_NAME_FORMAT = "\\\"" + capture(anythingBut("\\\""), COMPONENT_NAME_GROUP_NAME) + "\"";
     private static final String STEREOTYPE_FORMAT = "(?:<<" + capture(anythingBut("<>")) + ">>\\s*)";
     private static final Pattern STEREOTYPE_PATTERN = Pattern.compile(STEREOTYPE_FORMAT);
-
     private static final String ALIAS_GROUP_NAME = "alias";
     private static final String ALIAS_FORMAT = "\\s*(?:as \"?" + capture("[^\" ]+", ALIAS_GROUP_NAME) + "\"?)?";
 
     private static final String COLOR_FORMAT = "\\s*(?:#" + anyOf("\\w|/\\\\-") + "+)?";
-
+    private static final String COMPONENT_TYPE_GROUP_NAME = "componentType";
+    private static final String COMPONENT_TYPE_FORMAT = capture("component", COMPONENT_TYPE_GROUP_NAME) + "?";
```

Comment:
this is optional and lib should support both
```
[Comp1] <<..c1..>>
```
and
```
component [Comp1] <<..c1..>>
```
Currently it support only first option.